src/less/overwrites/_popover.less

```diff
@@ -13,51 +13,19 @@
 //
 // Styleguide 18.1
 
-//Color States are deprecated, please remove them soon
-
-.popover-variant(has-success; @state-success-bg; 5px solid @state-success-bg; #fff);
-.popover-variant(has-info; @state-info-bg; 5px solid @state-info-bg; #fff);
-.popover-variant(has-warning; @state-warning-bg; 5px solid @state-warning-bg; #fff);
-.popover-variant(has-error; @state-danger-bg; 5px solid @state-danger-bg; #fff);
-
-
-
-/* EXTENDED THE DEFAULT POPUP TO HAVE DIFFERENT STATES */
-.popover-variant(@state; @background; @border; @color) {
-
-  .popover{
-    &.@{state}{
-      background: @background;
-      color: @color;
-
-      &.top {
-        > .arrow:after{
-        border-top-color: @background;
-        }
-      }
-      &.bottom {
-        > .arrow:after{
-        border-bottom-color: @background;
-        }
-      }
-      &.left {
-        > .arrow:after{
-        border-left-color: @background;
-        }
-      }
-      &.right {
-        > .arrow:after{
-        border-right-color: @background;
-        }
-      }
-
-
-      .popover-title{
-        background: darken(@background, 5%);
-        color: @color;
-        border-bottom: @border;
-      }
-    }
+.popover {
+  color:#fff;
+  border: none;
+  border-radius: 0;
+  background-clip:border-box;
+  font-size: 15px;
```

Comment:
Please add a variable in the base variable set for this change as by default the popover font-size depends on @font-size-base.